br/tests/br_rawkv/run.py

```diff
@@ -99,14 +117,20 @@ def _exit_with_error(self, error):
 
 def main():
     args = parse_args()
-    tester = rawkvTester(args)
+
+    tester = rawkvTester(args.br, args, storage_path_suffix="basic")
     tester.test_rawkv()
 
+    tester_fp = rawkvTester(args.br_test, args, storage_path_suffix="fp", failpoints=FAILPOINT_TIKV_REGION_ERROR)
```

Comment:
can we set this failpoint in each test case directly? so we don't need to maintain global failpoint test variables.

Reply:
I will fix it later.